### PR TITLE
fix: pointer events, and add mesh name to the raycast result

### PR DESCRIPTION
**Pull request by kuruk-mm**
feat: add mesh name to the raycast result

Fix https://github.com/decentraland/godot-explorer/issues/293
Fix https://github.com/decentraland/godot-explorer/issues/63
Fix https://github.com/decentraland/godot-explorer/issues/359

### CODE REVIEW

File: rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs [hunk 1]
```diff
@@ -939,80 +939,69 @@ impl INode for SceneManager {
             &current_pointer_raycast_result,
         );
 
-        let should_update_tooltip = !GodotDclRaycastResult::eq_key(
```
Comment (leanmendoza) [inline rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs:942]: Is there any possibility of keeping this?

File: rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs [hunk 2]
```diff
@@ -939,80 +939,69 @@ impl INode for SceneManager {
...
                         }
                     }
                 }
             }
-
-            self.set_pointer_tooltips(tooltips);
-            self.base.emit_signal("pointer_tooltip_changed".into(), &[]);
         }
 
+        self.set_pointer_tooltips(tooltips);
```
Comment (leanmendoza) [inline rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs:1002]: If this will change every frame, maybe a signal is not necessary?

**Review by leanmendoza — APPROVED** [review]
It looks good. Only a question about removing the same pointer event check.

File: rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs (hunk 1 above)
Comment (kuruk-mm) [reply rust/decentraland-godot-lib/src/scene_runner/scene_manager.rs:942]: The distance changes on every frame and the eq_key just compare if the scene and entity are the same than the last one

**Pull request merged**